starter-core/src/main/resources/pom.xml

```diff
@@ -16,6 +16,11 @@
         </repository>
     </repositories>
     <dependencies>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-params</artifactId>
+            <version>5.9.0</version>
```

Comment:
This isn't needed. `micronaut-test-bom` imports `junit-bom`, which defines `junit-jupiter-params`

Reply:
resolved via https://github.com/micronaut-projects/micronaut-starter/pull/1433/commits/98fa17f537832b3f100bd87849a7afc3bb516112